prow/tide/blockers/blockers.go

```diff
@@ -70,16 +73,56 @@ func (b Blockers) GetApplicable(org, repo, branch string) []Blocker {
 }
 
 // FindAll finds issues with label in the specified orgs/repos that should block tide.
-func FindAll(ghc githubClient, log *logrus.Entry, label, orgRepoTokens string) (Blockers, error) {
+func FindAll(ghc githubClient, log *logrus.Entry, label string, orgRepoTokensByOrg map[string]string, splitQueryByOrg bool) (Blockers, error) {
+	queries := make(map[string]string, len(orgRepoTokensByOrg))
+	for org, query := range orgRepoTokensByOrg {
+		if splitQueryByOrg {
+			queries[org] = blockerQuery(label, query)
+		} else {
+			if queries[""] != "" {
+				queries[""] += " "
+			}
+			queries[""] += blockerQuery(label, query)
```

Comment:
Seems to me that we should simply use a set right away on `!splitQueryByOrg`, then you don't need to deal with first element, then split it the full string, then make a string from it again...

Reply:
The code below doesn't know anything about github apps auth or not, it just executes all the queries in the map. This means we can not use different data structures in `qureries`, depending on with auth mechanism is used.

I've updated the code to always store the query in a `set.String`, let me know what you think

Reply:
Your update is good! Sorry for not being clear enough.